RiotSharp/StaticDataEndpoint/StatsStatic.cs

```diff
@@ -8,135 +8,330 @@ namespace RiotSharp.StaticDataEndpoint
     public class StatsStatic
     {
         internal StatsStatic() { }
-
+        
+        /// <summary>
+        /// Flat Armor value for the object.
+        /// </summary>
```

Comment:
could you remove this commit as it's already been merged?

Reply:
You should be able to do so with a `git rebase origin/develop` if `origin` is the name of this remote.

Reply:
You'll have to excuse my lack of knowledge, but I used that command to no avail. the remote is named origin. Any advice?

Reply:
Probably causing more problems than I'm solving :)